examples/shader/shader_prepass.rs

```diff
@@ -212,30 +224,51 @@ impl Material for PrepassOutputMaterial {
     fn alpha_mode(&self) -> AlphaMode {
         AlphaMode::Blend
     }
+
+    #[cfg(target_arch = "wasm32")]
```

Comment:
That's what I originally tried, but for some reason, the webgl feature is not enabled in examples. I agree this isn't ideal but since it's only an example I think it's ok. The actual functionality uses the webgl feature.

Reply:
Ok.